pinot-core/src/test/java/org/apache/pinot/core/indexsegment/mutable/MutableSegmentImplTest.java

```diff
@@ -89,6 +93,11 @@ public void testMetadata() {
     SegmentMetadata expectedSegmentMetadata = _immutableSegment.getSegmentMetadata();
     Assert.assertEquals(actualSegmentMetadata.getTotalDocs(), expectedSegmentMetadata.getTotalDocs());
 
+    // assert that the last indexed timestamp is close to what we expect
+    long actualTs = _mutableSegmentImpl.getSegmentMetadata().getLastIndexedTimestamp();
```

Comment:
Do you also want to test if last ingestion time matches the default metadata time?